org.eclipse.lsp4e/src/org/eclipse/lsp4e/outline/SymbolsLabelProvider.java

```diff
@@ -119,7 +120,7 @@ public Image getImage(Object element) {
 		if (element == null){
 			return null;
 		}
-		if (element == LSSymbolsContentProvider.COMPUTING) {
+		if (element == LSSymbolsContentProvider.COMPUTING || element instanceof PendingUpdateAdapter) {
```

Comment:
Hi @ghentschke , 

I have read the class but I still do not understand the change. Which code is creating objects of this type in the context of LSP4E?

Reply:
It comes from `org.eclipse.ui.progress.DeferredTreeContentManager.getChildren(Object)` which is used to calculate the symbols for the Project Explorer tree asynchronously:
```java
	public Object[] getChildren(final Object parent) {
		IDeferredWorkbenchAdapter element = getAdapter(parent);
		if (element == null) {
			return null;
		}
		PendingUpdateAdapter placeholder = createPendingUpdateAdapter();
		startFetchingDeferredChildren(parent, element, placeholder);
		return new Object[] { placeholder };
	}
```